docs/snippets/examples/automatic-expire.py

```diff
@@ -0,0 +1,3 @@
+gb = GB()
+gb.foreach(lambda x: execute('EXPIRE', x['key'], 3600))
+gb.register('*', readValue=False)
```

Comment:
mode='sync'?

Reply:
I guess it is correct-er to do that. Fixed.